cleanrepo/CleanRepo/Program.cs

```diff
@@ -778,7 +727,7 @@ private static List<(string, List<string>)> GetSolutionFiles(string startDirecto
         foreach (var slnFile in dir.EnumerateFiles("*.sln", SearchOption.AllDirectories))
         {
             SolutionFile solutionFile = SolutionFile.Parse(slnFile.FullName);
-            List<string> projectFiles = solutionFile.ProjectsInOrder.Select(p => Path.GetDirectoryName(p.AbsolutePath).ToLowerInvariant()).Distinct().ToList();
+            List<string> projectFiles = solutionFile.ProjectsInOrder.Select(p => Path.GetDirectoryName(p.AbsolutePath)).Distinct().ToList();
```

Comment:
Do you need a proper `InvariantCultureIgnoreCase` here for comparison? Seems to me you could have duplicates which is why it was doing `ToLowerInvariant` to get rid of those duplicates via `Distinct`. However, these are the projects in a solution, how could you have two of the same? So probably ignore this note 😁